pyproject.toml

```diff
@@ -122,4 +122,21 @@ CXXFLAGS = "-Wno-error=stringop-overflow -Wno-pragmas"
 SKBUILD_CMAKE_BUILD_TYPE = "MinSizeRel"
 
 [tool.cibuildwheel.windows.environment]
-SKBUILD_CMAKE_BUILD_TYPE = "MinSizeRel"
\ No newline at end of file
+SKBUILD_CMAKE_BUILD_TYPE = "MinSizeRel"
+
+[[tool.cibuildwheel.overrides]]
+# Trigger the build & validation of the python stubs for certain platforms.
+# The test command acts as a kind of "post-build" callback where it's possible
+# for the stub-generator to import the freshly-built wheel.
+# There are two entry-points which are designed to call generate_stubs.py through
+# this test command:
+# - `make pystubs` is called during local development to generate the
+#   stubs and copy them into the git repo to be committed and reviewed.
+# - in CI, the cibuildwheel action is used to validate that the stubs match what
+#   has been committed to the repo.
+select = "cp311-manylinux_*64"
+test-requires = "mypy~=1.15.0 stubgenlib~=0.1.0"
```

Comment:
I decided use this instead of a PEP 723 script for `generate_stubs.py` because cibuildwheel creates a venv for testing with the OpenImageIO wheel pre-installed, and using `uv run` would create a new ad-hoc venv.